docs/operations/ksm.md

```diff
@@ -0,0 +1,90 @@
+# KSM Management
+
+Kernel Samepage Merging ([KSM](http://www.linux-kvm.org/page/KSM))
+allows de-duplication of memory. KSM tries to find identical Memory Pages and merge
+those to free memory.
+>Further Information:
+>- [KSM (Kernel Samepage Merging) feature](https://www.thomas-krenn.com/en/wiki/KSM_(Kernel_Samepage_Merging)_feature)
+>- [Kernel Same-page Merging (KSM)](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/virtualization_tuning_and_optimization_guide/chap-ksm)
+
+## Enabling KSM through KubeVirt CR
+KSM can be turned on in the nodes by `spec.configuration.ksmConfiguration` in the KubeVirt CR.\
```

Comment:
The backslash at the end just shows as-is in the page. Did you mean double-space to line-break?